### DISCUSSION TITLE
No metric when configuring monitors

### DISCUSSION THREAD

**Discussion by arend-melissant-tnt** [uptrace/uptrace#360] [Q&A]
I am running uptrace self hosted.

As of 1.7.0-rc.1 I am unable to select any metric when adding or editing monitors, Is there some new config added that I am missing?
1.7.0-beta.1 was working fine

when editing current montitors ther metric has a 'chip' which states 'deleted'. On the metric dropdown there are no items. I can use the filter above that, and it shows all the metric groups, but the dropdown for metric selection is empty.

All metrics do show up on the Metrics TAB

**Comment by vmihailenco** [comment]
It seems to be working here. You can try to check Uptrace logs and browser console logs (F12).

I would also try upgrading to v1.7.0-rc.3 and [reset the data](https://uptrace.dev/get/debugging.html#resetting-clickhouse-database).

**Reply by Arend-melissant** [reply]
Thanks for the response.

I have upgraded to rc3 and this fixed the issue

Arend
